backend/src/test/java/com/staccato/moment/repository/MomentRepositoryTest.java

```diff
@@ -107,12 +107,12 @@ void findAllByMemoryIdOrderByVisitedAt() {
         Moment moment3 = momentRepository.save(MomentFixture.create(memory, LocalDateTime.of(2024, 1, 10, 23, 21)));
 
         // when
-        List<Moment> moments = momentRepository.findAllByMemoryIdOrderByVisitedAt(memory.getId());
+        List<Moment> moments = momentRepository.findAllByMemoryIdOrderByVisitedAtDesc(memory.getId());
 
         // then
         assertAll(
                 () -> assertThat(moments.size()).isEqualTo(3),
-                () -> assertThat(moments).containsExactlyInAnyOrder(moment1, moment2, moment3)
+                () -> assertThat(moments).containsExactlyInAnyOrder(moment3, moment2, moment1)
```

Comment:
순서 검증이 필요하기 때문에 메서드가 바뀌어야 할 것 같습니다.

```suggestion
                () -> assertThat(moments).containsExactly(moment3, moment2, moment1)
```

Reply:
> 만약 날짜가 같다면 어떤 순서로 가져올지 궁금해서 테스트를 짜서 확인해봤는데, DB에 저장된 순서(아마 createdAt)대로 꺼내오는 것 같네요.

테스트 공유해주세욥!!!🤩

Reply:
> 그렇게 된다면 두 번째 정렬 조건으로 CreatedAtDesc가 추가될텐데, 이에 대해 어떻게 생각하시나요?

메서드 네이밍이 길어져서 가독성이 좋지 않다고 생각하여 JPQL로 변경했습니다. 확인하고 피드백 주세요:)